quickbooks_desktop_endpoint.rb

```diff
@@ -79,12 +79,13 @@ class QuickbooksDesktopEndpoint < EndpointBase::Sinatra::Base
 
       Persistence::Settings.new(config).setup
 
-      generate_and_add_guid
+      already_has_guid?
```

Comment:
usually the ? indicates that the method will return true or false for some kind of conditional... so it would make more sense to use it in the unless already_has_guid? line... but fine by me.

Reply:
I was trying to set the @already_has_guid variable before we do anything by calling the function. And then call the actual @ variable later?

Reply:
But I don't really fully understand memoization - I just know I need it (or something like it) here.

Reply:
you can then kill this line.